### ISSUE TITLE
build document fail

### ISSUE THREAD

**Issue by yiyuezhuo** [jstat/jstat#195]
I'm work on windows and I follow `Windows users have two options`.Anyway I don't know what the "same location" refer to, So I install them as default way.

I can build `jstat.js` and `jstat.min.js`, but it can't work on `doc`. I got this
![2](https://cloud.githubusercontent.com/assets/12798270/15984906/8cb90f12-300d-11e6-802a-394e74fa0053.png)

**Comment by trevnorris** [comment]
yeah. that part of the readme is ancient. I should really just make all that a node script. assigning to myself.

**Comment by yiyuezhuo** [comment]
@trevnorris I have watch the project for long time, and I feel a little careless for activting issues and pull requests. Thus, as a statistic undergrade, I would like to join jstat organization to help future developing. Can I get it?

**Comment by jamescgibson** [comment]
I think I might've been the over-clever person who introduced the `which` dependency - is that why this is failing? It has been a long, long time since I have used Windows.

**Comment by trevnorris** [comment]
@jamescgibson Not really sure. Been a year since I looked at it.
